platforms/android/src/jp/wizcorp/phonegap/plugin/WizAssets/WizAssetManager.java

```diff
@@ -197,4 +198,34 @@ public void deleteFolder(String filePath) {
             Log.e(TAG, "Delete file error -- " + e.getMessage(), e);
         }
     }
+
+    // If we detect a data structure created by plugin version <= 5.0.0 we want to clean up
+    private void checkForMigration() {
+        String deprecatedDbLocation = that.getCacheDir().getAbsolutePath() + File.separator + "assets.db";
```

Comment:
Could it be possible to put `assets.db` and `assets` in two constant at the top of this function? Something such as `DATABASE_NAME_AT_VERSION_500` and `TABLE_NAME_AT_VERSION_500`.